packages/components/src/pages/InstallGuide.vue

```diff
@@ -202,7 +157,11 @@ export default {
         this.selectedProject = project;
       })
       .$on('status-jump', (pageIndex) => {
-        this.jumpToIndex(pageIndex);
+        if (pageIndex === 2) {
+          this.$root.$emit('open-navie');
```

Comment:
In the PR description, can you create a section that describes:

1) New events that the IDE will need to handle
2) New data that the IDE will need to provide

Reply:
Good point, done :+1: